connectors/invidious.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+import BaseConnector from 'content/base-connector';
+import Utils from 'content/utils';
+import _ from 'underscore';
+
+new class extends BaseConnector {
+    constructor() {
+        super();
+        this.name = 'Invidious';
+        this.prefix = '/com/invidious';
```

Comment:
This should be hostname in reverse order, eg. `/us/invidou`. Looks ugly but I think we should be consistent in prefix scheme.